### DISCUSSION TITLE
1.2.1

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#9950] [General]
* Fix intermittent localhost forwarding issues (solves #9921)
* Use /mnt/wslg for /run/user (solves #9913)
* Fix invalid handle close on wslservice.exe exit

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/1.2.1'>1.2.1</a>.</em>

**Comment by crramirez** [comment]
Remember to mark this version as the latest release
